torchgeo/datasets/biomassters.py

```diff
@@ -0,0 +1,344 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""BioMassters Dataset."""
+
+import os
+from typing import Any, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import download_url, extract_archive, percentile_normalization
+
+
+class BioMassters(NonGeoDataset):
+    """BioMassters Dataset for Aboveground Biomass prediction.
+
+    Dataset intented for Aboveground Biomass (AGB) prediction
+    over Finnish forests based on Sentinel 1 and 2 data with
+    corresponding target AGB mask values generated by Light Detection
+    and Ranging (LiDAR).
+
+    Dataset Format:
+
+    * .tif files for Sentinel 1 and 2 data
+    * .tif file for pixel wise AGB target mask
+    * .csv files for metadata regarding features and targets
+
+    Dataset Features:
+
+    * 13,000 target AGB masks of size (256x256px)
+    * 12 months of data per target mask
+    * Sentinel 1 and Sentinel 2 data for each location
+    * Sentinel 1 (specify channels here) available for each month
```

Comment:
Either specify channels or remove band list

Reply:
Do the filenames of the downloaded files tell you? If not, maybe just remove "(specify channels here)" so our docs don't look like TODO statements